src/lib/components/scheduling/SongAvailibilityTimeline.svelte

```diff
@@ -117,13 +114,11 @@
 							{#if availability.available}
 								{getTimeString(availability.startTime)} -
 								{getTimeString(availability.endTime)}
+							{:else if availability.reason}
+								Unavailable: <br />
+								{availability.reason}
 							{:else}
-								{#if availability.reason}
-									Unavailable: <br />
-									{availability.reason}
-								{:else}
-									Unavailable
-								{/if}
+								Unavailable
```

Comment:
worth considering om misschien de 'Unavailable' altijd te tonen en de ': <br/> {availability.reason}' conditional te maken. Maar dit is ook leesbaar dus ik laat t aan jou

Reply:
Denk dat het duidelijker is om niet een conditional te hebben die begint met een `: <br />`